website/package.json

```diff
@@ -13,9 +13,8 @@
     "build-readme": "./scripts/build-readme.mjs"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.75",
-    "@docusaurus/preset-classic": "2.0.0-alpha.75",
-    "@docusaurus/theme-search-algolia": "^2.0.0-alpha.75",
```

Comment:
This isn't needed. The Algolia theme comes with `preset-classic`—the documentation wasn't super clear about this before.

Reply:
Ah, nice, good to know.